src/content/app/genome-browser/components/drawer/components/sequence-view/DrawerSequenceView.scss

```diff
@@ -27,6 +27,10 @@
   grid-area: aside-top;
 }
 
+.blastSequenceButton {
+  --blast-sequence-button-margin: 0 0 0 25px;
```

Comment:
I am confused as to how we are initialising this css variable and how it is working.
It feels the wrong file to initialise it. Should we not initialise it in `src/shared/components/blast-sequence-button/BlastSequenceButton.scss` where it feels like the root css file. 
And how is it working, how it is setting the margin-left to 25px? should we not have something like this: 
`margin: var(--blast-sequence-button-margin, 0 0 0 25px)`

Reply:
`BlastSequenceButton.scss` says: I am going to use the variable `--blast-sequence-button-margin` if it's available, or I will fall back to a default value.

This allows us to set the CSS variable anywhere up the DOM tree relative to BlastSequenceButton, and BlastSequenceButton will see it.

In this case, DrawerSequenceView component wants to move BlastSequenceButton around.

Reply:
Now that I think of it, maybe `BlastSequenceButton` doesn't need the margin variable. Maybe, since it never defines a margin itself, it is sufficient for BlastSequenceButton to accept a css class that would have the margin defined.

_(This is different from the `--radio-label-font-weight` though. There, the radio component really needs to define the font-weight of its label.)_

Reply:
Removed the `--blast-sequence-button-margin` variable.